pkg/nbcontracthelper/helper.go

```diff
@@ -0,0 +1,11 @@
+package nbcontracthelper
+
+const (
+	defaultLinuxUser = "azureuser"
+)
+
+func GetDefaultLinuxAdminUsername() string {
+	// var abc nbc.ApiServerConfig
```

Comment:
what is var abc?

Reply:
sorry this was generated by copilot. I should have removed it 🤣